### PR TITLE
Add a regular expression in order to avoid reading bvh erros in the s…

### PR DESCRIPTION
**Pull request by CristianNajeraL**
…plit_bvh.py process

### CODE REVIEW

**Comment by dpacgopinath** [comment]
Thanks for the catch, @CristianNajeraL ! I have left a comment inline.

File: fairmotion/tasks/clustering/split_bvh.py [hunk 1]
```diff
@@ -25,8 +26,9 @@ def main(args):
     os.makedirs(args.output_folder, exist_ok=True)
     for root, _, files in os.walk(args.folder, topdown=False):
         for filename in tqdm.tqdm(files):
-            filepath = os.path.join(root, filename)
-            split_bvh(filepath, args.time_window, args.output_folder)
+            if re.match(r'(\w+).bvh', filename):
+                filepath = os.path.join(root, filename)
+                split_bvh(filepath, args.time_window, args.output_folder)
```
[the remark this answers is not in the document]
Comment (dpacgopinath) [reply fairmotion/tasks/clustering/split_bvh.py:31]: Sorry for the delay, I had forgotten to hit "submit review" :|

**Comment by dpacgopinath** [comment]
Hi @CristianNajeraL , thanks for the PR -- do you want to incorporate the suggested changes so we can merge this to master?

**Comment by CristianNajeraL** [comment]
> Hi @CristianNajeraL , thanks for the PR -- do you want to incorporate the suggested changes so we can merge this to master?

@dpacgopinath I'm running out of time in other projects. I will try to do this weekend but if you want to do it, go ahead!

**Pull request closed without merging**